app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,23 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by_id session[:user_id]
```

Comment:
Dynamic is not recommended to use now, you can use find_by and pass the key and value as a parameter